r2/r2/lib/media.py

```diff
@@ -312,6 +312,16 @@ def _scrape_media(url, autoplay=False, maxwidth=600, force=False,
             print "%s made a bad secure media obj for url %s" % (scraper, url)
             secure_media_object = None
 
+        # If thumbnail can't be found, attempt again using _ThumbnailOnlyScraper
+        # This should fix bugs that occur when embed.ly caches links before the thumbnail is available
```

Comment:
Minor style nit: we try to follow PEP-8-ish style restrictions, which includes making lines no longer than 79 characters, so this comment would need to be wrapped onto another line: https://github.com/reddit/styleguide/blob/master/python/pep-0008.rst#maximum-line-length